challenge/src/components/Pages/About.jsx

```diff
@@ -1,15 +1,30 @@
 import React from 'react'
-import RandomImage from '../RandomImage'
+import JojoStand from '../JojoStand'
 
-const About = (props) => {
-    const page = props.currentpage
-    const pictureStyling = {
-        marginTop: "5%",
-        marginBottom: "5%",
-        marginRight: "5%",
-        marginLeft: "5%"
+class About extends React.Component {
+    constructor(props){
+      super(props);
+      this.state = {
+        standsVisibility: [
+          true,true,true,true
+        ],
+        length: 4
+      }
     }
-    return (
+    snap = () => {
+      var oldStands = this.state.standsVisibility.slice(0, this.state.length/2);
+      var replacedStands = this.state.standsVisibility.slice(this.state.length/2);
+      for(var i = 0; i < replacedStands.length; i++){
+        replacedStands[i] = false;
+      }
+      var newStands = oldStands.concat(replacedStands);
```

Comment:
if you do the the clone thing you won't have to concat the old and replaced. which makes things cleaner/make a little more sense.